src/main/java/com/projectg/geyserupdater/common/util/GeyserApi.java

```diff
@@ -0,0 +1,32 @@
+package com.projectg.geyserupdater.common.util;
+
+import com.google.gson.Gson;
+import com.projectg.geyserupdater.common.pojo.Root;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class GeyserApi {
```

Comment:
Different name than the class in Geyser's actual API please - maybe `GeyserDownloadsApi`